packages/hackathon/src/scenes/UsersScene/components/HackerList.tsx

```diff
@@ -80,9 +81,9 @@ export const HackerList: FC<HackerListProps> = ({
   const rows = hackers
     .slice(startIdx, startIdx + PAGE_SIZE)
     .map((hacker, idx) => (
-      <DataTableCell key={idx}>
+      <DataTableItem key={idx} id={idx.toString()}>
```

Comment:
Why is the id needed now?

Does id map to the html id? If there are multiple data table instances in a scene using this pattern for id do we end up with duplicate ids? Prefix with "Hacker-" maybe. I'm not sure and I dont know the purpose of this at this time

Reply:
`id` is used to power the `select` features of `DataTable` (demo at: https://components.looker.com/storybook/?path=/story/datatable--select).

Admittedly, in an ideal world we'd be able to use `key` but React doesn't let us have access to that for reasons I can't entirely argue with.

Reply:
Oops... hit enter too fast. 

I suppose we could make it optional if there was a strong argument against it but I'd like `select` to "just work" when it's enabled and we'd have to build for a case where some rows are missing IDs and decide how to handle that if it wasn't required.

Reply:
Thanks for the explanation. Wondering is selectId might be a better name to avoid confusion